kaos/models.py

```diff
@@ -41,13 +41,16 @@ class SatelliteInfo(SavableModel, DB.Model):
         platform_name:      The name of the satellite
         orbit_segments:     Time segments that the ephemeris records fall within
         orbit_records:      Satellite ephemeris records
+        orbit_q_max:        The maximum distance from the earth center to the satellite position
     """
     __tablename__ = 'SatelliteInfo'
 
     platform_id = DB.Column(DB.Integer, primary_key=True)
     platform_name = DB.Column(DB.String(50), nullable=False)
     orbit_segments = DB.relationship("OrbitSegments", backref='satellite_info', lazy=True)
     orbit_records = DB.relationship("OrbitRecords", backref='satellite_info', lazy=True)
+    orbit_q_max = DB.Column(DB.Float, nullable=True)
```

Comment:
`orbit_q_max` would only make sense to someone who's really familiar with the algorithm -- we should change this to something more descriptive, like `max_altitude`